new-log-viewer/src/components/Layout.css

```diff
@@ -0,0 +1,32 @@
+/* Layout.css */
```

Comment:
Any reason we want to add the file name in the stylesheet?
```suggestion
```

Reply:
I thought it is a tradition to add these on top of css files. Maybe I used too much ChatGPT and their generated css code all starts with these file name headers XD.